docs/security/authentication/auth-provider-compatibility.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Authentication provider compatibility
+description: Compatibility of authentication providers differ between Octopus Server and Octopus Cloud.
+position: 0
+---
+
+Octopus ships with a number of authentication providers. The support for these providers differ between Octopus Server and [Octopus Cloud](/docs/octopus-cloud/index.md). Some authentication providers only work with Octopus Server, whilst others only work with Octopus Cloud. This page describes the compatibility of these providers in Octopus.
+
+## Login support {#login-support}
+
+The following table shows login support for each authentication provider in Octopus Server and Octopus Cloud:
+
+|                                       | Octopus Server     | Octopus Cloud   |
+|---------------------------------------|:------------------:|:---------------:|
+| Username and Password                 | :white_check_mark: | :white_check_mark: **\*** |
+| Active Directory Authentication       | :white_check_mark: | :x:&nbsp;&nbsp; |
+| Azure Active Directory Authentication | :white_check_mark: | :white_check_mark: **\*** |
+| GoogleApps Authentication             | :white_check_mark: | :white_check_mark: **\*** |
+| Okta Authentication                   | :white_check_mark: | :x:&nbsp;&nbsp; |
+| Octopus ID                            | :x: | :white_check_mark:&nbsp;&nbsp; |
+| Guest Login                           | :white_check_mark: | :white_check_mark:&nbsp;&nbsp; |
+| GitHub                                | :x: | :white_check_mark: **\*** |
+
+**Note:** Entries marked with **\*** are supported via [Octopus ID](octopusid-authentication.md).
+
+## External groups and roles support {#external-groups-and-roles}
+
+Octopus allows [external groups and roles](/docs/security/users-and-teams/external-groups-and-roles.md) to be added as members of Teams in Octopus. The following table shows which authentication providers support this in Octopus Server and Octopus Cloud:
+
+|                                       | Octopus Server     | Octopus Cloud   |
```

Comment:
I love the table for Login support above. I'm having a little trouble coming around on this table. The entire `Octopus Cloud` column is a no, and *most* of the entries for Octopus Server are also no so it just makes it feel.... not very well supported? I do see the value in having a consistent table that matches the table above to describe the authentication features. In this case though, could we consider just going with a line or two saying something like:

```
Octopus Server supports external groups and roles using Active Directory, Azure AD, and Okta. Octopus Cloud does not support this feature at this time.
```

Reply:
I know what you mean. The balance here is not showing cloud at all vs being clear on what is supported and what isn't. Adding an option of mapping groups to the other table didn't feel right as it would require 2 more columns and I felt that would make it cluttered. So I think that although it shows a lot of not supported on Cloud that is accurate

Reply:
Sorry didn't see your other sentence. I'd like to guage other opinions and we'll see where the land lies 😄

Reply:
@pstephenson02 - we could go with a hybrid. What about one column for the second table just indicating Octopus Server support with a sentence stating Octopus Cloud doesn't support external groups at this time?

Reply:
@harrisonmeister That sounds like a good compromise 👍

Reply:
@pstephenson02 - mulled this point overnight and I'm going to leave it as-is for now. I think the benefits of the grid outweigh any negative association for Cloud. I think the transparency is important, and in addition, it allows us to easily flip a :x: into a  :heavy_check_mark: if and when we add support for Groups for any of the other auth providers.